voip_utils/sip.py

```diff
@@ -168,6 +226,80 @@ def __init__(self, sdp_info: SdpInfo) -> None:
         """Set up SIP server."""
         self.sdp_info = sdp_info
         self.transport = None
+        self._outgoing_calls: dict[str, int] = {}
+
+    def outgoing_call(
+        self, source: SipEndpoint, destination: SipEndpoint, rtp_port: int
+    ):
+        """Make an outgoing call from the given source endpoint to the destination endpoint, using the rtp_port for the local RTP port of the call."""
+        if self.transport is None:
+            _LOGGER.warning("No transport for outgoing VOIP calls")
```

Comment:
No need to log the warning. If the code calling this method thinks it's worth logging, it can log the error.
```suggestion
```